fe/fe-core/src/main/java/org/apache/doris/statistics/AnalysisInfo.java

```diff
@@ -182,14 +182,19 @@ public enum ScheduleType {
     @SerializedName("usingSqlForPartitionColumn")
     public final boolean usingSqlForPartitionColumn;
 
+    /**
+     * Used to store the newest partition version of tbl when creating this job.
+     */
+    public final long tblUpdateTime;
```

Comment:
do not need presist?

Reply:
No, this info would finally save in TableStatsMeta

Reply:
add comment to this variable, it is very confuse